2024-02 newsletter (#747)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2024-02-14-newsletter.md

---
layout: post
title: February 2024 Newsletter
editor: Peter Heywood
slug: 2024-02-14-newsletter
date: 2024-01-14T17:00:00Z+0100
tags:
category:
link:
description:
type: text
---

## The University of Sheffield Research Software Engineering Community Newsletter February 2024

> Welcome to the January 2024 newsletter for the research software community at The University of Sheffield, featuring

Reviewer: ```suggestion
Welcome to the February 2024 newsletter for the research software community at The University of Sheffield, featuring
```

#### Wider RSE network events

- N8 CIR Digital Research Infrastructure Retreat 2024
  - 2024-03-18 to 2024-03-22 in Manchester (remote attendance available)
  - Accommodation, breakfast and lunch are included for delegates attending in person
  - [Website](https://n8cir.org.uk/events/digital-research-infrastructure-retreat-2024/)
  - [Registration (by 2024-03-12 23:45:00)](https://web.cvent.com/event/f446f6ec-a9a2-4f9c-a997-4fe2c5cec428/summary?locale=en)
- Institute of Computing for Climate Science (ICCS) ReproHack
  - 2024-03-12 1pm to 6pm, Remote (Zoom). Registrations close 2024-02-29.
  -[Website](https://www.reprohack.org/event/27/)
- RSE Midlands meeting in Warwick on 2024-03-07
  - The event is fully booked, but a wait list is available.
  - [Website](https://rse-midlands.github.io/docs/event-7th-march-2024/)
  - [Registration (Wait list)](https://pretix.eu/rsemidlands/2024/)
- Carpentry Connect Heidelberg 2024
  - 2024-11-12 to 2024-11-14, Heidelberg, Germany
  - [Website](https://carpentries.org/blog/2024/01/announcing-cchd24/)
- Royal Statistical Society Talk: - Medical Statistician Apprenticeships and Integrating R into GXP Medical Research

>   - 2024-02-28 1600-17:30

Reviewer: ```suggestion
  - 2024-02-28 16:00-17:30
```

  - [Website](https://www.software.ac.uk/workshop/collaborations-workshop-2024-cw24)
- RSECon24
  - 2024-09-03 to 2024-09-05, Newcastle, UK + Remote
  - Call for submissions: March - May 2024
  - Call for reviewers: March - April 2024
  - Call for volunteers: March - April 2024
  - Registration: Mid April - June 2024
  - [Website](https://rsecon24.society-rse.org/)
- UseR! 2024 Conference
  - 2024-07-08 to 2024-07-11, Salzburg, Austria + Remote
  - [Website](https://user2024.r-project.org)
- An introduction to [Octopus.ac](https://www.octopus.ac/)
  - 2024-03-28 Online
  - Octopus.ac is a free, UKRI funded publishing model that forgoes the traditional research paper in favour of a series of “micro publications” that more closely model how research is conducted.
  During this session we’ll invite you to share your perceptions of Octopus and welcome feedback from researchers of all disciplines
  - [Event Registration](https://events.teams.microsoft.com/event/f6858265-3a51-4dac-b6bd-09e478c3cd2d@48f9394d-8a14-4d27-82a6-f35f12361205)

### Open Research

> - FAIR Faculty seminar series is now all scheduled on my development (requires login)

Reviewer: ```suggestion
- FAIR Faculty seminar series is now all scheduled on myDevelopment (requires login)
```

  - Various dates in February, grouped by Faculty.

>   - [Mydevelopment](https://mydevelopment.csod.com/ui/lms-learner-search/search?pageNumber=1&query=FAIR%20faculty%20seminars)

Reviewer: ```suggestion
  - [myDevelopment](https://mydevelopment.csod.com/ui/lms-learner-search/search?pageNumber=1&query=FAIR%20faculty%20seminars)
```

- [International Science Council - Open Science Round-up January 2024](https://council.science/current/blog/open-science-round-up-january-2024/ )
- [Ibrahim, Hazem, et al. "Google Scholar is manipulatable." arXiv preprint arXiv:2402.04607 (2024).](https://arxiv.org/abs/2402.04607)
- [JOSSCast: Open Source for Researchers](https://blog.joss.theoj.org/2024/01/introducing-josscast). A new podcast from Journal of Open Source Software with new episodes every Thursday.
- [Code for Thought: [EN] Patterns in Science And Medicine - Veronika Cheplygina](https://codeforthought.buzzsprout.com/1326658/14338771-en-patterns-in-science-and-medicine-veronika-cheplygina)
- [Code for Thought: [EN] ByteSized RSE: Easy Does It - with easybuild - Jörg Saßmannshausen](https://codeforthought.buzzsprout.com/1326658/14361557-en-bytesized-rse-easy-does-it-with-easybuild-jorg-sassmannshausen)

<!-- ### Tutorial of the month -->

### Training

#### Deep Learning Demystified: Foundations for Non-Computer Scientists

> - 2024-02-22 13:00-17:00 and 2024-02-32 13:00-17:00

Reviewer: ```suggestion
- 2024-02-22 13:00-17:00 and 2024-02-23 13:00-17:00
```

Reviewer: One more typo!

Author: ta
